crates/flox/doc/flox-run.md

```diff
@@ -11,21 +11,58 @@ flox-run - run app from current project
 
 # SYNOPSIS
 
-flox [ `<general-options>` ] run [ `<options>` ] [ -- [ `<command args>` ... ] ]
+flox [ `<general-options>` ] run [ `<run-options>` ] [ -- [ `<command args>` ... ] ]
 
 # DESCRIPTION
 
 Run flake application from the requested package (or "installable").
 If not provided `flox` will prompt for you to select from the list of known packages.
+`flox run` uses `nix run` under the hood to execute the so-called installable. Which makes it possible to run 
+packages from any Nix flake that exposes the `app` attribute in its outputs.
 
-# OPTIONS
+# EXAMPLES
 
-```{.include}
-./include/general-options.md
-./include/development-options.md
+## Running applications in the current working directory
+
+If `flox run` is called without any arguments, it will ask the user which application they want to use.
+
+```console
+$ flox run
+? Select a packageapp for flox run  
+> flox
+  flox-bash
+  nix-editor
+[↑ to move, enter to select, type to filter]
+```
+
+If `flox run` is called with an agrument, it will try to run that one instead.
+
+```
+$ flox run cowsay -- 'Moo!'
```

Comment:
Where is cowsay coming from? Would it be more obvious to do `flox run nix-editor` or something?

Reply:
I'm also having difficulty with this one. Because, when I call it like `flox run cowsay`, I'm assuming there's an underlying `flake.nix` and `flox.nix` with cowsay in there somewhere. Actually, that probably doesn't even work. 🤔

How would you tackle this?

Reply:
I think I'd give an example in the same context as above (the flox repo). And maybe clarify that it's coming from a local repo